src/Umbraco.Web.UI.Client/src/less/components/umb-grid.less

```diff
@@ -219,16 +221,18 @@
     top: 50%;
     left: 50%;
     transform: translate(-50%, -50%);
-    color: @ui-action-type;
+    color: @ui-action-disgrete-type;
 }
 
 .umb-grid .cell-tools-add.-bar {
     display: block;
-    background: @gray-10;
     text-align: center;
     padding: 5px;
-    border: 1px dashed @gray-7;
+    border: 1px dashed @ui-action-disgrete-border;
```

Comment:
The term `disgrete` is misspelled everywhere, this should be `discrete`

Reply:
Should install a spell-checker for my IDE.
I have created a issue on this matter:
https://github.com/umbraco/Umbraco-CMS/issues/4648